.taskcluster.yml

```diff
@@ -0,0 +1,227 @@
+        then: ${event.ref[11:]}
+        else: 'dev'
+      else: 'dev'
+
+    taskboot_image: "mozilla/taskboot:0.1.6"
+  in:
+    - taskId: {$eval: as_slugid("bot_checks")}
+      provisionerId: aws-provisioner-v1
+      workerType: github-worker
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '1 hour'}
+      extra:
+        github:
+          events:
+            - pull_request.opened
+            - pull_request.reopened
+            - pull_request.synchronize
+            - push
+      payload:
+        maxRunTime: 3600
+        image: python:3-alpine
+        command:
+          - sh
+          - -lxce
+          - "apk add git --quiet &&
+            git clone --quiet ${repository} /src && cd /src/bot && git checkout ${head_rev} -b checks &&
+            pip install . && pip install -r requirements-dev.txt &&
+            flake8 && pytest -v"
+      metadata:
+        name: Code Review Bot checks
+        description: Check python code (unit tests & linting)
+        owner: bastien@mozilla.com
+        source: https://github.com/mozilla/code-review
+
+    - taskId: {$eval: as_slugid("frontend_build")}
+      provisionerId: aws-provisioner-v1
+      workerType: github-worker
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '1 hour'}
+      extra:
+        github:
+          events:
+            - pull_request.opened
+            - pull_request.reopened
+            - pull_request.synchronize
+            - push
+      payload:
+        maxRunTime: 3600
+        image: node:11-alpine
+        command:
+          - sh
+          - -lxce
+          - "apk add git lcms2-dev bash libpng-dev autoconf build-base --quiet &&
+            git clone --quiet ${repository} /src && cd /src/frontend && git checkout ${head_rev} -b build &&
+            npm install && npm run build"
+        artifacts:
+          public/frontend:
+            expires: {$fromNow: '2 weeks'}
+            path: /src/frontend/build
+            type: directory
+      metadata:
+        name: Code Review Frontend build
+        description: Build web single page application
+        owner: bastien@mozilla.com
+        source: https://github.com/mozilla/code-review
+
+    - taskId: {$eval: as_slugid("bot_build")}
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '1 hour'}
+      provisionerId: aws-provisioner-v1
+      workerType: releng-svc
+      dependencies:
+        - {$eval: as_slugid("bot_checks")}
+      payload:
+        capabilities:
+          privileged: true
+        maxRunTime: 3600
+        image: "${taskboot_image}"
+        env:
+          GIT_REPOSITORY: ${repository}
+          GIT_REVISION: ${head_rev}
+        command:
+          - taskboot
+          - build
+          - --image
+          - mozilla/code-review
+          - --tag
+          - "${channel}"
+          - --tag
+          - "${head_rev}"
+          - --write
+          - /bot.tar
+          - bot/Dockerfile
```

Comment:
Is this going to write an image with two tags?

Reply:
Yeap

Reply:
There may be a bug on taskboot, as i only see the hashes on the repo, not the `testing` tag

Reply:
Maybe we need something like https://github.com/mozilla/task-boot/pull/38 for the build command too?

Reply:
I think it should be in the `push-artifact` command to iterate over tags and push all of them https://github.com/mozilla/task-boot/blob/master/taskboot/push.py#L52 instead of just one